recipes/DateTimeRange/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "DateTimeRange" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/DateTimeRange-{{ version }}.tar.gz
+  sha256: 47bc10e5df23bb1f0d3ee493a26cb8d1d99e179884aabff62d3e51033b6a73ce
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - releasecmd
+    - pip
+  run:
+    - python >=3.6
+    - typepy >=1.0.0,<2
+    - python-dateutil >=2.4.2,<3
+
+test:
+  imports:
+    - datetimerange
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/thombashi/DateTimeRange
+  summary: DateTimeRange is a Python library to handle a time range. e.g. check whether a time is within the time range, get the intersection of time ranges, truncating a time range, iterate through a time range, and so forth.
```

```suggestion
  summary: DateTimeRange is a Python library to handle a time range. 
  description: |
      Check whether a time is within the time range, get the intersection of time ranges, 
      truncating a time range, iterate through a time range, and so forth.
```

Summary is really supposed to be a 1 line thing